apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx

```diff
@@ -677,7 +693,7 @@ export const TOTPAuthenticator: React.FunctionComponent<TOTPProps> = (
                                                 />
                                             )
                                         }
-                                        content={ t(translateKey + "hint") }
+                                        content={ "Add new QR code" }
```

Comment:
Shall we localize this?

Reply:
Fixed with b5e82899c9c6b76d2ba7e0e0e5e41d10436e5bc0